### PR TITLE
S3t2 setup sanitiser and validator and unittests

### PR DESCRIPTION
**Pull request by Carinav**

### CODE REVIEW

File: tests/Services/Validators/EmailValidatorTest.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    {
+        $result = EmailValidator::validateEmail('Example@example.com');
+        $this->assertEquals($result, 'Example@example.com');
+    }
+
+    public function testValidateEmailFailure()
+    {
+        $result = EmailValidator::validateEmail('Example@examplecom');
+        $this->assertEquals($result, false);
+    }
```
Comment (henryppercy) [inline tests/Services/Validators/EmailValidatorTest.php:20]: Should we do a malformed test here as well?

File: src/Services/Sanitisers/StringSanitiser.php [hunk 2]
```diff
@@ -4,5 +4,17 @@
...
-
-}
\ No newline at end of file
+    /**
+     * @param string $validateData
+     * @return string
+     */
+    public static function sanitiseString(?string $validateData): string
+    {
+        if ($validateData === null) {
```
Comment (henryppercy) [inline src/Services/Sanitisers/StringSanitiser.php:13]: Do we need a check for null?

**Review by henryppercy — CHANGES_REQUESTED** [review]

File: tests/Services/Sanitisers/StringSanitiserTest.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+{
+    public function testSanitiseStringSuccess()
+    {
+        $str = '    <h1>Hello World!</h1>';
+        $result = StringSanitiser::sanitiseString($str);
+        $expected = 'Hello World!';
+        $this->assertEquals($result, $expected);
+    }
+
+    public function testSanitiseStringSuccessNull()
```
Comment (henryppercy) [inline tests/Services/Sanitisers/StringSanitiserTest.php:18]: And the corresponding test

File: tests/Services/Validators/EmailValidatorTest.php (hunk 1 above)
Comment (TherealGuah) [reply tests/Services/Validators/EmailValidatorTest.php:20]: Extra test added testing array insertion into validator

**Pull request merged**